packages/core/src/i18n.ts

```diff
@@ -1,7 +1,7 @@
 import { interpolate, UNICODE_REGEX } from "./context"
 import { isString, isFunction } from "./essentials"
 import { date, number } from "./formats"
-import { compileMessage } from "./compile"
+import { compileMessage } from "@lingui/core/compile"
```

Comment:
That's very important change. If we would access this with relative path, tree shaker would not be able to eliminate this import. 

Because of package import, tree shaker first looks into package's pakage.json, see `{sideEffects: false}` and then remove it from the bundle.